desktop/dev-environments/create-dev-env.md

```diff
@@ -101,23 +107,24 @@ Docker then clones the repository with your specified branch or tag.
 
 1. From **Dev Environments** in Docker Dashboard, select **Create**. The **Create a Dev Environment** dialog displays.
-2. Select **Get Started** and then copy your Git subfolder link into the **Enter the Git Repository** field with **Existing Git repo** as the source.
-3. Select **Continue**.
-
-    This clones the Git code inside a volume, determines the best image for your dev environment, and opens VS Code inside the dev environment container.
+2. Select **Get Started**.
+3. Optional: Provide a name for you dev environment.
+4. Select **Existing Git repo** as the source and then paste the link of your Git repo subfolder into the field provided.
+5. Choose your IDE. You can choose either:
+    - **Visual Studio Code**. The Git repository is cloned into a Volume and attaches to your containers. This allows you to develop directly inside of them using Visual Studio Code.
+    - **Other**. The Git repository is cloned into your chosen local directory and attaches to your containers as a bind mount. This shares the directory from your computer to the container, and allows you to develop using any local editor or IDE.
+6. Select **Continue**.
 
-4. Hover over the container and select **Open in VS Code** to start working. You can also open a terminal in VS Code, and use Git to push or pull code to your repository, or switch between branches and work as you would normally.
-
-5. To launch the application, run the command `make run` in your terminal. This opens an http server on port 8080. Open [http://localhost:8080](http://localhost:8080) in your browser to see the running application.
+To launch the application, run the command `make run` in your terminal. This opens an http server on port 8080. Open [http://localhost:8080](http://localhost:8080) in your browser to see the running application.
 
 ## Launch from a local folder
 
 1. From **Dev Environments** in Docker Dashboard, select **Create**. The **Create a Dev Environment** dialog displays.
-2. Select **Get Started** and then choose **Local Folder** as the source.
-3. Next to **Select your local directory** field, select **Select** to open the root of the code that you would like to work on.
-4. Select **Continue**.
-
-    This detects the main language of your local folder, creates a dev environment using your local folder, and bind-mounts your local code in the dev environment. It then opens VS Code inside the dev environment container.
+2. Select **Get Started**.
+3. Optional: Provide a name for your dev environment.
+4. Choose **Local directory** as the source.
+5. Select **Select** to open the root directory of the code that you would like to work on.
+    A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```

Comment:
Suggestion: add an empty line here to create a separate paragraph for the description part of the instruction

```suggestion
   
   A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```